packages/devtools_app/test/flutter/timeline_screen_test.dart

```diff
@@ -23,8 +23,44 @@ import 'wrappers.dart';
 
 void main() {
   TimelineScreen screen;
+  TimelineScreenBodyState state;
   FakeServiceManager fakeServiceManager;
 
+  Future<void> buildTimelineScreen(
```

Comment:
naming suggestion: pumpTimelineScreen

To distinguish building a widget tree and actually placing it into the test app.